news/2022-04-27-new-featured-artist-MUZZ.md

```diff
@@ -0,0 +1,107 @@
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Spectrum.mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Draining Atlantic
+
+If you didn't have enough of [Secretpipe](https://osu.ppy.sh/users/2208964) yet, here's [another osu! marathon](https://osu.ppy.sh/beatmapsets/479671).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Draining%20Atlantic.mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Lost Forever
+
+Try out [this osu! marathon](https://osu.ppy.sh/beatmapsets/600276) hosted by [Mir](https://osu.ppy.sh/users/8688812)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Lost%20Forever.mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Children of Hell
+
+Here's another [osu! marathon](https://osu.ppy.sh/beatmapsets/452965) hosted by [Secretpipe](https://osu.ppy.sh/users/2208964)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Children%20of%20Hell.mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Timberwolf
+
+[Another osu! marathon](https://osu.ppy.sh/beatmapsets/419291). Do we really need to say who mapped it?
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Timberwolf.mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Calling Out (feat. KG & Skyelle)
+
+Test your limits with [this loved map](https://osu.ppy.sh/beatmapsets/392041) hosted by [elchxyrlia](https://osu.ppy.sh/users/1722835)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Calling%20Out%20(feat.%20KG%20%26%20Skyelle).mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Break Away (feat. Priority One)
+
+On the opposite side of the difficulty spectrum, try [this easier osu! marathon](https://osu.ppy.sh/beatmapsets/789823) hosted by [Peter](https://osu.ppy.sh/users/8623835)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Break%20Away%20(feat.%20Priority%20One).mp3" type="audio/mpeg">
+</audio>
+
+### MUZZ - Crescendo (feat. MYLK)
+
+If circles aren't your thing, enjoy [this osu!catch map](https://osu.ppy.sh/beatmapsets/526032) hosted by [WildOne94](https://osu.ppy.sh/users/3482692)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/261/Songs/MUZZ%20-%20Crescendo%20(feat.%20MYLK).mp3" type="audio/mpeg">
+</audio>
+
+---
+
+Here's [**MUZZ**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/261). You know what to do.
+
+In case you missed [our Stonebank announcement](https://osu.ppy.sh/home/news/2022-04-06-new-featured-artist-stonebank), we're now working with **Monstercat** to bring talents beloved by the osu! community to our Featured Artists collection. Check the [**Monstercat** listing](https://osu.ppy.sh/beatmaps/artists/255) for a continually updated list of *Monstercat* tunes!
```

Comment:
```suggestion
In case you missed [our Stonebank announcement](https://osu.ppy.sh/home/news/2022-04-06-new-featured-artist-stonebank), we're now working with **Monstercat** to bring talents beloved by the osu! community to our Featured Artists collection. Check the [**Monstercat** listing](https://osu.ppy.sh/beatmaps/artists/255) for a continually updated list of **Monstercat** tunes!
```